app/src/main/java/org/piramalswasthya/cho/utils/DateTimeUtil.kt

```
@@ -257,7 +260,7 @@ class DateTimeUtil {
                 val outputDateFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
                 val date = inputDateFormat.parse(dateString)
                 return outputDateFormat.format(date);
-            } catch (e: Exception){
+            } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)